app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("NotVaccinatedError")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("OutdatedVaccineError")
+
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("NotWearingMaskError")
```

Comment:
Read the checklist. Also you can add  visitor's name to the messages

![image](https://user-images.githubusercontent.com/104218550/220340150-d7787795-6e5a-4bf1-ac32-699ec4185b88.png)